### DISCUSSION TITLE
How to handle missings datas when using Auto-regression ?

### DISCUSSION THREAD

**Discussion by theoeiferman** [ourownstory/neural_prophet#335] [Q&A - get help using NeuralProphet] [answered]
Indeed I get this error : 
ValueError: cannot reindex from a duplicate axis

Also is there a way to say to the model :" use Auto-regression with Hours" , is freq='H' when applying fit to the data enough ?

Thank you in advance,

**Comment by ourownstory** [comment, marked as answer]
Hi @theoeiferman 

> ValueError: cannot reindex from a duplicate axis

Can you please provide the minimal code for us to replicate your issue?

> Also is there a way to say to the model :" use Auto-regression with Hours" , is freq='H' when applying fit to the data enough ?

if you like to use AR(12), set n_lags=12

when working with hourly data `freq='H'` is correct
